website-page1.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <link href="style.css" rel="stylesheet">
+    <title>Muna Fidow Website</title>
+    <style="background-color: darkolivegreen;"></style>
+  </head>
+  <body>
+    <h1>Muna Fidow Story</h1>
+    <div class="flex-container">
+    <div><img src="images/muna.jpg"WIDTH="300" HEIGHT="300" alt="Muna"/></div>
```

Comment:
do you need these divs? `div` should be a last resort as a html tag, but these also seem a little redundant anyway